example_kubeflow_pipelines/openvino/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenVINO workflow demo with TensorFlow slim models optimization and evaluation
+
+
+This pipeline integrates three tasks:
+
+- [TensorFlow slim models generator](container/tf_slim.md) - it creates slim models which are considered as exemplary to 
+demonstrate OpenVINO model optimization and execution
+- [OpenVINO model optimizer](container/mo.md) - it converts the model 
+from TensorFlow or other frameworks into Intermediate Representation format. 
+It is one time operation. The result can be used in the inference execution using Inference Engine.
+- [Inference Engine prediction](container/predict.md) - it is running the inference evaluation based on 
+input and label data in numpy files
+
+The pipeline can be used to assess the results of model optimization in terms of inference execution time and 
+classification accuracy. 
+
+During the pipeline execution there are generated several artifacts.
+ 
+In the path set by `tf-export-dir` parameter:
+- Graph definition in protobuffer format
+- Frozen graph created based on imported checkpoint
+
+In the path set by `generated-model-dir` parameter:
+- TensorFlow model in a frozen saved_model format. It can be consumed by TensorFlow based inference scripts or TensorFlow 
+serving. This format can't be used for additional training because all variables are changed to constants.
+- Intermediate Representation of the optimized model created based on the exported TF saved model. It can be used to run 
+inference operation 
+using OpenVINO Inference Engine or to be imported by OpenVINO Model Server
+- TensorFlow even file which can be used to visualize the graph in tensorboard
+
+`generated-model-dir` value should be pointed to a shared location like google storage, persistent storage or NFS 
+so it could be passed between components. 
+
+![demo pipeline1](demo_pipeline1.png)
+![demo pipeline2](demo_pipeline2.png)
+![demo pipeline3](demo_pipeline3.png)
+## Parameters
+
+- model_name = slim model name which match the allowed topologies from 
+[slim repo](https://github.com/tensorflow/models/blob/master/research/slim/nets/nets_factory.py)
+- num_classes = how many classes is defined in the model. For slim models it could be `1000` or `1001` depending on the model
+- checkpoint_url = URL to the compressed checkpoint for the [pre-trained models](https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models)
+-batch_size - batch size for the generated IR model
+- export_dir - location for storing auxiliary model formats
+- generated_model_dir - location for storing pipeline results - google storage path. If set to a numerical subfolder 
```

Comment:
I updated the documentation to avoid this problem. It requires missing path to prevent overloading of the content.
It shouldn't be required to include secrets if the cluster is in the same GCP project and cluster has enabled 'Allow full access to all Cloud APIs'.
I think we could include support for secrets in next version.